fuzz/fuzz_targets/fuzz_process_deep.rs

```diff
@@ -0,0 +1,240 @@
+use honggfuzz::fuzz;
```

Comment:
Performance may degrade for sufficiently large input indices. I recommend putting a cap on the for loop from 0..=input.len() to something fixed.